backend/app/store/image/image.go

```diff
@@ -76,6 +88,10 @@ type submitReq struct {
 
 // NewService returns new Service instance
 func NewService(s Store, p ServiceParams) *Service {
+	p.commitTTL = p.EditDuration * 15 / 10  // Commit call on every 1.5 * EditDuration
```

Comment:
how 15/10 is easier to understand? even comment saying 1.5x

Reply:
this has to be integers and not float value, I can't put `* 1.5` there unfortunately.

Reply:
ah, ok.